tasks/pull_date_format.js

```diff
@@ -0,0 +1,169 @@
+            var outSuffix = existingOutputText.substr(outObjEndIndex);
+            var objText = existingOutputText.substr(outObjStartIndex, outObjEndIndex - outObjStartIndex);
+            var outObj;
+            try {
+                outObj = eval('a=' + objText);
+            }
+            catch(e) {
+                throw new Error(locale + '--' + objText);
+            }
+
+            if(isNew) {
+                outObj.name = locale;
+                outObj.dictionary = {};
+                outObj.format = {};
+            }
+
+            var format = outObj.format = (outObj.format || {});
+
+            format.days = wcDataObj.dayNames;
+            format.shortDays = wcDataObj.dayNamesShort;
+            format.months = wcDataObj.monthNames;
+            format.shortMonths = wcDataObj.monthNamesShort;
+            format.date = translateDateFormat(wcDataObj.dateFormat);
+            // leave out `dateTime` and `time` fields. We could include them as cmd line args later
+            // for now these will get inherited from english.
+
+            var outStr = JSON.stringify(outObj, null, 4)
+                .replace(/'/g, '\\\'') // escape single quotes
+                .replace(/"([A-Za-z]\w+)":/g, '$1:') // unquote simple identifier keys
+                .replace(/([^\\])"/g, '$1\'') // replace unescaped doublequotes with singlequotes
+                .replace(/\\"/g, '"'); // unescape escaped doublequotes
+
+            outStr = shortenArrays(outStr);
+
+            common.writeFile(pathToOutput, outPrefix + outStr + outSuffix, function() {
+                console.log('ok pull_date_format ' + locale);
+            });
+        });
+    });
+}
+
+/*
+ * Turn HTML entities into unicode - for explicit compatibility with SVG
+ * There are just a few in world-calendars that aren't already in unicode anyway.
+ */
+var entityMap = {
+    ccedil: '\u00e7',
+    auml: '\u00e4',
+    '#x10C': '\u010c'
+};
+function cleanHTMLEntities(s) {
+    return s.replace(/&([^;]+);/g, function(entity, entityID) {
+        var outChar = entityMap[entityID];
+        if(!outChar) {
+            throw new Error('no unicode character listed for ' + entity);
+        }
+        return outChar;
+    });
+}
+
+/*
+ * Convert world-calendars date format ('dd.mm.yyyy') to d3 ('%d.%m.%Y')
+ */
+function translateDateFormat(wcDateFormat) {
+    var out = wcDateFormat
+        .replace('yyyy', '%Y')
+        // bosnian (bs) uses 2-digit year in world-calendars, but I don't want to continue that!
+        .replace('yy', '%Y')
+        .replace('mm', '%m')
+        .replace('dd', '%d')
+        // gujarati (gu) uses short month name in world-calendars - keep it?
+        .replace('M', '%b');
+
+    return out;
+}
+
+/**
+ * Either put the whole array on one line, split it into 2 lines,
+ * or leave it as is (one line per item), whatever we can fit into 90 characters.
+ * A trailing comma may still be after the 90-character limit
+ */
+function shortenArrays(s) {
```

Comment:
That's brilliant :sparkles: 

Maybe down the road, we could factor this out and use it on our `test/image/mocks/` JSONs to make them a bit more readable.